cdhweb/projects/admin.py

```diff
@@ -3,11 +3,11 @@
 from mezzanine.core.admin import DisplayableAdmin
```

Comment:
same deal with this file as the other `admin.py`, it can just be deleted. I'll set up the admin management for `ProjectRelatedLink`s as part of #182.

Reply:
addressed by 4a10cd4